### ISSUE TITLE
Не авторизуется

### ISSUE THREAD

**Issue by Yevgen16** [kkuryshev/mosenergosbyt#6]
Не проходит авторизация. Может чего-то в Мосэнергосбыте поменялось?

**Comment by YrKirillov** [comment]
Да, не авторизуется, хотя если дебажить то ответ от сервера есть.
Вот такую ошибку показывает:
{'err_code': 101, 'err_id': 'MES1-WSOWS02.interrao.ru/jsonlk_comu, 2023-08-22 14:39:26', 'err_text': 'Required jsonGate parameter not found or unsupported action', 'metaData': {'responseTime': 0.0}, 'success': False}

**Comment by shrlnm** [comment]
@YrKirillov Тут похоже не ошибка авторизации.
Любой err_code кроме 201 приводит к выдаче сообщения об ошибке авторизации

https://github.com/kkuryshev/mosenergosbyt/blob/2ea213ead6bba15e801675e9d90b6efa8b536016/mosenergosbyt/session.py#L24-L31

а в вашем случае 'err_code': 101

Проблема похоже в том, что этот код работает только с одним плагином bytProxy.

https://github.com/kkuryshev/mosenergosbyt/blob/2ea213ead6bba15e801675e9d90b6efa8b536016/mosenergosbyt/meter.py#L111-L117

Как я понял, у разных поставщиков услуг разные плагины, у меня в экаунте зарегистрировано два адреса. По одному адресу для водяных счетчиков используется как раз bytProxy, а по другому адресу используется другое название плагина и другой набор json параметров, в результате у меня тоже выдается "ошибка авторизации"

Покопавшись немного с сайтом mosenergosbyt бросил эту затею. У  mosobleirc вроде более понятный API, хотя туда подтягиваются не все счетчики. Советую посмотреть в сторону mosobleirc
